src/commands/up.rs

```diff
@@ -164,9 +173,9 @@ impl UpCommand {
             .context("Could not canonicalize working directory")?;
 
         let resolved_app_source = self.resolve_app_source(&app_source, &working_dir).await?;
-
-        let trigger_cmds = trigger_command_for_resolved_app_source(&resolved_app_source)
-            .with_context(|| format!("Couldn't find trigger executor for {app_source}"))?;
+        let mut trigger_cmds =
+            trigger_command_for_resolved_app_source(resolved_app_source.trigger_types()?)
+                .with_context(|| format!("Couldn't find trigger executor for {app_source}"))?;
```

Comment:
Rather than doing this (potentially) twice, maybe this can be moved inside `if self.help` below and the second call to `trigger_command_for_resolved_app_source` below can move outside of that `if` block?